assemblies/server/src/main/resources-filtered/etc/custom.properties

```diff
@@ -137,7 +137,7 @@ org.osgi.framework.system.packages.extra= \
  org.slf4j.*; version="${slf4j.version}", \
  org.springframework.dao; version="${spring.version}", \
  org.springframework.security.*; version="${spring-security.version}", \
- org.yaml.snakeyaml; version="${snakeyaml.version}", \
+ org.yaml.snakeyaml; version="${safeyaml.version}", \
```

Comment:
Shouldn't the groupId be "org.hitachivantara"?

Reply:
Not really.